kvbc/cmf/categorized_kvbc_msgs.cmf

```diff
@@ -92,7 +92,10 @@ Msg BlockData 2001 {
 Msg RawBlockData 2005 {
     fixedlist uint8 32 parent_digest
     CategoryInput updates
-    map string optional fixedlist uint8 32 category_root_hash
+    map string fixedlist uint8 32 block_merkle_root_hash
+    map string fixedlist uint8 32 versioned_root_hash 
+    map string map string fixedlist uint8 32 immutable_root_hashes
```

Comment:
I think I didn't get the comment about optional hashes. Could you please explain it?

Reply:
I think it's better to include the hashes only in case they are really calculated i.e. not nullopt.
It will save data on the wire on ST and is more compact.

Reply:
Thanks! Yes, I got it. I was thinking we are using these for reconstruction on ST, but we are not.